collections-project2_2021-02-08.sql

```diff
@@ -0,0 +1,65 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.33)
+# Database: collections-project2
+# Generation Time: 2021-02-08 15:28:02 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table collections
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `collections`;
+
+CREATE TABLE `collections` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `food` varchar(30) DEFAULT 'null',
+  `colour` tinyint(2) unsigned NOT NULL,
+  `size-rating` tinyint(2) unsigned NOT NULL,
+  `healty-rating` tinyint(2) unsigned NOT NULL,
+  `image-path` varchar(50) DEFAULT NULL,
```

Comment:
- only comment which is non-critical is the field names in db could do with _ instead of -